src/engine.cc

```diff
@@ -58,6 +58,9 @@ const OptionId kUciChess960{
     "Castling moves are encoded as \"king takes rook\"."};
 const OptionId kShowWDL{"show-wdl", "UCI_ShowWDL",
                         "Show win, draw and lose probability."};
+const OptionId kStrictUciTiming{"strict-uci-timing", "StrictTiming",
+                                "The UCI host compensates for lag, and waits "
+                                "for the 'readyok' reply before sending 'go'."};
```

Comment:
'and only start timing when go arrives'?

Reply:
I reworded it a bit.